src/interactivity/win32/Clipboard.cpp

```diff
@@ -319,6 +300,40 @@ void Clipboard::CopyTextToSystemClipboard(const TextBuffer::TextAndColor& rows,
     globalHandle.release();
 }
 
+// Routine Description:
+// - Copies the given string onto the global system clipboard in the sepcified format
+// Arguments:
+// - stringToCopy - The string to copy
+// - lpszFormat - the name of the format
+void Clipboard::CopyToSystemClipboard(std::string stringToCopy, LPCWSTR lpszFormat)
+{
+    const size_t rtfSize = stringToCopy.size() + 1; // +1 for '\0'

```

Comment:
I'd probably rename this too
```suggestion
    const size_t cbData = stringToCopy.size() + 1; // +1 for '\0'
```

Just to not mix up "HTML" and "RTF" in the same function

Reply:
Done